row.py

```diff
@@ -81,9 +100,215 @@ def convert_pdf_to_pil(pdf_as_bytes):
+            maxRadius=max_rad,
+        )
+
+        if detected_circles is None:
+            circle_count = 0
+        else:
+            circle_count = len(detected_circles[0])
+
+        logging.debug("Circles: %4i", circle_count)
+        logging.debug("Multiplier: %5.4f", ratio_multiplier)
+        logging.debug("Fudge: %7i", fudge_value)
+        logging.debug("Radius: %6d-%d", min_rad, max_rad)
+        logging.debug("Inset: %6i", inset)
+
+        count_down -= 1
+
+    return export_circles_from_image(
+        detected_circles,
+        output_path,
+        item_path,
+        img,
+        height,
+        width,
+        inset,
+    )
+
+
+def ocr_images(directory):
+    """OCRs all image files (.jpg) in directory
+
+    Args:
+
+         directory (Path): The directory where the images are located
+
+     Returns:
+         pd.dataframe: A dataframe with OCR results
+    """
+    #: create initial dataframe to work with
+    working_df = pd.DataFrame({"Number": [], "Filename": [], "Parcel": []})
+    working_df = working_df.astype({"Number": int, "Filename": object, "Parcel": object})
+
+    #: read all files in the directory
+    ocr_num = 0
+    file_list = list(directory.glob("*.jpg"))
+    for ocr_file in file_list:
+        ocr_count = ocr_file.stem.rsplit("_", 1)[1]
+
+        logging.debug("working on: %s", ocr_file.stem)
+        ocr_full_path = directory / ocr_file
+        #: read in image from bytes
+        byte_img = ocr_full_path.read_bytes()
+        ocr_img = cv2.imdecode(np.frombuffer(byte_img, dtype=np.uint8), 1)  # 1 means flags=cv2.IMREAD_COLOR
+
+        #: convert image to grayscale
+        ocr_gray = cv2.cvtColor(ocr_img, cv2.COLOR_BGR2GRAY)
+
+        ocr_text = pytesseract.image_to_string(ocr_gray, config=OCR_CONFIG)
+        ocr_text = ocr_text.replace("\n", "").replace("\t", "").replace(" ", "")
+        df_new = pd.DataFrame(
+            {
+                "Number": [int(ocr_count)],
+                "Filename": [ocr_file],
+                "Parcel": [ocr_text],
+            }
+        )
+
+        working_df = pd.concat([working_df, df_new], ignore_index=True, sort=False)
+        ocr_num += 1
+
+    return working_df
+
+
+def export_circles_from_image(circles, out_dir, file_path, cv2_image, height, width, inset_distance):
+    """export detected circles from an image as jpegs to the out_dir
+
+        Args:
+            circles (array): Circle locations returned from cv2.HoughCircles algorithm
+            out_dir (Path): The output directory for cropped images of detected circles
+            file_path (Path): The location of the image file
+            cv2_image (numpy.ndarray): The image as a numpy array
+            height (number): The height of original image
+            width (number): The width of original image
+            inset_distance (number): The inset distance in pixels to aid image cropping
+
+    Returns:
+        list: a list of images and the count of images
```

Comment:
Isn't this just returning a list of the images and not the count?  Though the count can be determined by the length of the list.